superflore/repo_instance.py

```diff
@@ -35,10 +35,11 @@ def __init__(
         self.repo_url = repo_url.format(self.repo_owner, self.repo_name)
         self.repo_dir = repo_dir or self.repo_name
         # by default, start on master.
-        self.branch = from_branch or 'master'
+        self.from_branch = from_branch or 'master'
+        self.branch = self.from_branch
```

Comment:
I think I"m failing to see what this accomplishes?

Reply:
@allenh1 << It lets the PR base be specified by --upstream-branch instead of always being master:
    - If we're cloning from some upstream branch, diff should be against it too. So if --upstream-branch is specified, that needs to be the base when preparing the PR.